### DISCUSSION TITLE
Sonoffs with bistable relay

### DISCUSSION THREAD

**Discussion by cruiser2** [arendst/Tasmota#18370] [Q&A] [answered]
Sonoff seems to be very mute about the kind of relay (mono or bistable) they put into their devices. I even cannot find any information about this in their "Specifications" of products on their website. I own a POW320D, which has a bistable relay - but otherwise I am in the dark. Has anybody more info about products which use a bistable relay?

**Comment by sfromis** [comment, marked as answer]
What "more info" do you mean?

Based on the templates repository, these devices use the Tasmota support for bistable relays
https://templates.blakadder.com/sonoff_POWR320D
https://templates.blakadder.com/sonoff_THR320D
https://templates.blakadder.com/sonoff_THR320
https://templates.blakadder.com/lsc_smart_connect_3004200
https://templates.blakadder.com/sygonix_SY-4538254

**Reply by cruiser2** [reply]
Unfortunately not much. Basically just one device in slight variations. I suspect the SPM-4Relay might also have bistable relays - but I could not find any info by Sonoff. 
Thanks - I call this answered.
